apps/docs/src/app/core/component-docs/bar/examples/bar-default-example.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { RtlService } from '@fundamental-ngx/core';
+
+@Component({
+    selector: 'fd-bar-default-example',
+    templateUrl: './bar-default-example.component.html'
+})
+export class BarDefaultExampleComponent implements OnInit {
+
+    navigationArrow$: BehaviorSubject<string> = new BehaviorSubject<string>('navigation-left-arrow');
+
+    constructor(private rtlService: RtlService) { }
+
+    ngOnInit(): void {
+        if (this.rtlService) {
+            this.rtlService.rtl.subscribe(value => {
```

Comment:
You need to take care of that subscription, to avoid memory leak.
This can be simplified to one Observable usage.
Same for `BarHeaderExampleComponent`, `BarPageExampleComponent` and `BarPageResponsiveExampleComponent`

Reply:
also for all examples we can skip null check for rtl service because we controle when this service is injected